src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -214,10 +216,13 @@ export default function EnvironmentEditor({
         setScenes(copiedScenes);
     };
 
-    const onTransitionModalSubmit = async () => {
+    const onTransitionModalSubmit = async (transitions) => {
         setEditTransitionModalOpen(false);
 
-        // TODO: edit transition data upon edit
+        const envData = environment;
+        envData.transitions.data = transitions;
+        const response = await editScenario(envData, handleError);
+        setEnvironment(response.data);
     };
```

Comment:
Are we firing a request each time we change the transition order?

Reply:
When you "submit" the modal, then yes. I can add a quick check for if it's the same order?